ArcadiaParties.Data/Data/Seed.cs

```diff
@@ -0,0 +1,63 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using ArcadiaParties.Data.Models;
+using Newtonsoft.Json;
+
+
+namespace ArcadiaParties.Data.Data
+{
+    public class Seed
+    {
+        private const bool V = true;
+
+        public async static Task<Boolean> SeedData(DataContext context)
+        {
+            if (!context.Department.Any())
+            {
+                var depData = System.IO.File.ReadAllText("../ArcadiaParties.Data/Data/Depart.json");
+                var deps = JsonConvert.DeserializeObject<List<Department>>(depData);
+                foreach (var dep in deps)
```

Comment:
You can use `AddRangeAsync` instead of `for...each` cycle.

Reply:
Now using AddRangeAsync